src/compose/types/page-block/report.ts

```diff
@@ -0,0 +1,38 @@
+import { PageBlock, PageBlockInput, Registry } from './base'
+import { Apply, CortezaID, NoID } from '../../../cast'
+
+const kind = 'Report'
+
+interface Options {
+  reportID: string;
+  scenarioID: string;
+  blockID: string;
+  elementID: string;
+}
+
+const defaults: Readonly<Options> = Object.freeze({
+  reportID: NoID,
+  scenarioID: '',
+  blockID: '',
+  elementID: '',
```

Comment:
Should all of these have `NoID`?

Reply:
Yes,
I used '' up until we didn't have backend implementation for ID's. Since the thing I used was key and that was '' by default